content/source/docs/cloud/users-teams-organizations/single-sign-on/okta.html.md

```diff
@@ -48,3 +50,25 @@ Be sure to copy the metadata URL (from the final step of configuring Okta) befor
 6. Your Okta SSO configuration is complete and ready to [use](../single-sign-on.html#using-sso).
 
     ![sso-settings](../images/sso/settings-okta.png)
+
+## Configuration - Team Mapping (Okta)
```

Comment:
- Markdown headers should have blank lines after them. 
- The order of words in this header is somewhat confusing. How about switching them around a bit? 

```suggestion
## Team Mapping Configuration (Okta)

```

Reply:
I originally did this to keep consistency with the other sections, but I agree that it's not as clear.  I've updated the section name for clarity.